### PR TITLE
Add CI validation for the release drafter configuration

### PR DESCRIPTION
**Pull request by PierreBtz**
This is WIP to explore configuration validation by CI. It should address #21 .

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node('node') {
```
Comment (oleg-nenashev) [inline Jenkinsfile:1]: FTR https://github.com/jenkins-infra/documentation/blob/master/ci.adoc#node-labels
Node should be good enough if it includes the required tools

File: scripts/validateReleaseDrafterConfiguration.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+set -euo pipefail
+
+HERE="$( cd -P "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+RELEASE_DRAFTER_CONFIG="$HERE/../.github/release-drafter.yml"
+
+AJV_VERSION='3.2.0'
+
+pushd $TMPDIR || exit
+echo "Installing AJV"
+npm install -s -g "ajv-cli@${AJV_VERSION}"
```
Comment (oleg-nenashev) [inline scripts/validateReleaseDrafterConfiguration.sh:12]: Not a big overhead to download it each time., should be fine

File: scripts/validateReleaseDrafterConfiguration.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+HERE="$( cd -P "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+RELEASE_DRAFTER_CONFIG="$HERE/../.github/release-drafter.yml"
+
+AJV_VERSION='3.2.0'
+
+pushd $TMPDIR || exit
```
Comment (oleg-nenashev) [inline scripts/validateReleaseDrafterConfiguration.sh:10]: Is it needed on a single-shot agent?

File: scripts/validateReleaseDrafterConfiguration.sh (hunk 2 above)
Comment (PierreBtz) [reply scripts/validateReleaseDrafterConfiguration.sh:12]: The other approach would be to cook a Docker image and run the script through it using a docker node... Seems overkill just for one dependency. Maybe to consider if we start to have a whole set of tools to run.

File: scripts/validateReleaseDrafterConfiguration.sh (hunk 3 above)
Comment (PierreBtz) [reply scripts/validateReleaseDrafterConfiguration.sh:10]: Not really, but useful to run the script locally, before committing after changing the release drafter configuration.

**Comment by oleg-nenashev** [comment]
@PierreBtz do you think this is ready to go?

**Comment by PierreBtz** [comment]
@oleg-nenashev I don't know, I may be nitpicking but I would like to have https://github.com/release-drafter/release-drafter/pull/600 to directly depend on an officialy released schema instead of some commit in github. That being said the upstream PR is not moving so I'm happy to merge with the TODO and fix later when my upstream PR is merged.

**Comment by jetersen** [comment]
https://github.com/release-drafter/release-drafter/pull/600 is merged

**Comment by oleg-nenashev** [comment]
Cool

**Comment by PierreBtz** [comment]
@jetersen thanks a lot, is there any release planned soon for release drafter?

**Comment by jetersen** [comment]
The schema is read from master branch

**Comment by PierreBtz** [comment]
[schema store](https://www.schemastore.org/json/) points to a release tag. If I don't point to the same commit, it means that the IDE validation won't match the build validation.
If you don't have a release planned soon, it's ok, I'll keep pointing to a commit for now with a TODO.

**Comment by jetersen** [comment]
I'll look into releasing a new version later today.

**Comment by PierreBtz** [comment]
That's great! Thanks a lot :)

**Comment by jetersen** [comment]
😢 Schemastore changed the URLs... https://github.com/SchemaStore/schemastore/pull/1297

**Comment by PierreBtz** [comment]
MMhh I see, thanks for looking into it, take your time as far as this PR is concerned this is not urgent at all.

**Comment by PierreBtz** [comment]
I somehow missed the schema store reverted their change and is now properly using the master branch. This means we are all set here, this should be ready to merge @jetersen @oleg-nenashev

**Review by jetersen — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
No objections, thanks!
I am not sure the `node` label will always contain cURL, but it is quite plausible

**Pull request merged**